lib/cylc/gui/app_gcylc.py

```diff
@@ -3077,8 +3077,8 @@ def check_task_filter_buttons(self, tb=None):
                 box = ebox.get_children()[0]
                 try:
                     cb_ = box.get_children()[1]
-                except (ValueError, AttributeError):
-                    # ValueError: an empty box to line things up.
+                except (IndexError, AttributeError):
+                    # IndexError: an empty box to line things up.
                     # AttributeError: the name filter entry box.
```

Comment:
```
empty = []
_, a = empty  # AttributeError
a = empty[1]  # ValueError
```

TODO - we've made a bunch of these changes recently, to get rid of spurious variables. Better go back and check for corresponding changes to exceptions...

Reply:
Nasty, I'll see if I can find any more examples...

Reply:
I can't see any in #2318 or #2375.

Reply:
Thanks for checking!